crates/rolldown_error/src/events/ambiguous_external_namespace.rs

```diff
@@ -0,0 +1,44 @@
+use std::sync::Arc;
+
+use super::BuildEvent;
+use oxc::span::Span;
+
+use crate::{diagnostic::Diagnostic, types::diagnostic_options::DiagnosticOptions, EventKind};
+
+#[derive(Debug)]
+pub struct AmbiguousExternalNamespace {
+  pub importer: String,
+  pub importee: Vec<String>,
+  pub importer_source: Arc<str>,
+  pub importer_filename: String,
+  pub imported_specifier: String,
+  pub imported_specifier_span: Span,
+}
+
+impl BuildEvent for AmbiguousExternalNamespace {
+  fn kind(&self) -> EventKind {
+    EventKind::AmbiguousExternalNamespace
+  }
+
+  fn message(&self, _opts: &DiagnosticOptions) -> String {
+    format!(
+      r#""{}" re-exports "{}" from one of the modules {} (will be ignored)."#,
+      self.importer,
+      self.imported_specifier,
+      self.importee.iter().map(|v| format!(r#""{v}""#)).collect::<Vec<_>>().join(" and ")
```

Comment:
You need to consider the following scenarios: 
1. the  ambiguous **re-export** exists in two modules.
2. the ambiguous **re-export** exists in more than two modules.

Using a List with Commas and a Conjunction
The most common way to list items is by using commas to separate each item and using "and" (or "or") before the last item.

Example:

We need to buy apples, oranges, and bananas.

Reply:
Yes, I have considered it. Is it like below, I just need to optimize the separation?

https://github.com/rolldown/rolldown/blob/06ca7c787047e225e71524af5b2d6c3038eef0f4/crates/rolldown/tests/fixtures/errors/ambiguous_external_namespace/artifacts.snap#L15-L17

Reply:
that's not correct, when there is ambiguous **re-export** from three different modules, the message should be 
```bash
"a.js", "b.js" and "c.js"
```

Reply:
for two modules, it should be : 
```bash
"a.js" and "b.js"
```